server/src/controller/build.js

```diff
@@ -136,9 +136,12 @@ class BuildController {
         }
       })
       .orderBy(sort, direction)
-      .fetchAll()
+      .simplePaginate({
+        limit: pagination.perPage,
+        page: pagination.page
+      })
 
-    return builds.toJSON()
+    return builds
```

Comment:
Why did you removed `.toJSON()`? Could it be a mistake?

Reply:
It's not a mistake. `fetchAll` function returns a object `Collection` which has the function `toJSON()` to convert the result to json but `simplePaginate` returns a json response like this:
```javascript
{
  data: [
    { }, // object 1
    { },
    ...
  ],
 meta: {
    //pagination
  }